eras/shelley/impl/src/Cardano/Ledger/Shelley/PParams.hs

```diff
@@ -321,22 +335,30 @@ upgradeProposedPPUpdates ::
 upgradeProposedPPUpdates args (ProposedPPUpdates ppus) =
   ProposedPPUpdates $ upgradePParamsUpdate args <$> ppus
 
-ppMinFeeA :: EraPParams era => PParam era
-ppMinFeeA =
+ppMinFeeFactor :: EraPParams era => PParam era
+ppMinFeeFactor =
   PParam
     { ppName = "txFeePerByte"
-    , ppLens = ppMinFeeACompactL
-    , ppUpdate = Just $ PParamUpdate 0 ppuMinFeeACompactL
+    , ppLens = ppMinFeeFactorL
+    , ppUpdate = Just $ PParamUpdate 0 ppuMinFeeFactorL
     }
 
-ppMinFeeB :: EraPParams era => PParam era
-ppMinFeeB =
+ppMinFeeA :: EraPParams era => PParam era
+ppMinFeeA = ppMinFeeFactor
+{-# DEPRECATED ppMinFeeA "In favor of `ppMinFeeFactor`" #-}
+
+ppMinFeeConstant :: EraPParams era => PParam era
+ppMinFeeConstant =
```

Comment:
I don't quite see this as a great name, but I think consistency with cardano-api wins the argument here as well.
```suggestion
ppTxFeeFixed =
```